bitrise.yml

```diff
@@ -1,278 +1,67 @@
-    - SIMULATOR_PLATFORM: $SIMULATOR_PLATFORM # iOS Simulator
-    - OUTPUT_TOOL: xcpretty
-    - HEADLESS_MODE: "yes"
-    - RETRY_ON_FAIL: "yes"
-    after_run:
-    - _test
-    - check_exported_artifacts
-
-  manual_test-retry_after_fail:
+  sample:
     envs:
-    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git
-    - SAMPLE_APP_BRANCH: failing-ui-test
-    - OUTPUT_TOOL: xcpretty
     - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
     - BITRISE_SCHEME: ios-simple-objc
-    - SIMULATOR_DEVICE: iPhone 8 Plus
-    - SIMULATOR_PLATFORM: iOS Simulator
-    - HEADLESS_MODE: "yes"
-    - RETRY_ON_FAIL: "yes"
-    after_run:
-    - _test
-    - check_exported_artifacts
-
-  ci-objc_headless_mode:
-    envs:
-    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git
-    - SAMPLE_APP_BRANCH: master
-    - OUTPUT_TOOL: xcodebuild
-    - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
-    - BITRISE_SCHEME: ios-simple-objc
-    - SIMULATOR_DEVICE: iPhone 8 Plus
-    - SIMULATOR_OS_VERSION: "latest"
-    - SIMULATOR_PLATFORM: iOS Simulator
-    - HEADLESS_MODE: "yes"
-    - RETRY_ON_FAIL: "no"
-    after_run:
-    - _test
-    - check_exported_artifacts
-
-  ci-objc_headless_mode_xcpretty:
-    envs:
-    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git
-    - SAMPLE_APP_BRANCH: master
-    - OUTPUT_TOOL: xcpretty
-    - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
-    - BITRISE_SCHEME: ios-simple-objc
-    - SIMULATOR_DEVICE: iPhone 8 Plus
-    - SIMULATOR_OS_VERSION: "latest"
-    - SIMULATOR_PLATFORM: iOS Simulator
-    - HEADLESS_MODE: "yes"
-    - RETRY_ON_FAIL: "no"
-    after_run:
-    - _test
-    - check_exported_artifacts
-    - check_xcpretty_html_report
-
-  _script_wrapper_for_headless_failing_ui_test:
     steps:
     - script:
-        title: Start a failing workflow, wrapped in a script.
         inputs:
         - content: |-
             #!/bin/env bash
-            bitrise run ci-objc_headless_mode_failing_ui_test
-            if [ $? -ne 1 ] ; then
-              echo "Workflow was excepted to fail, exit code not 1."
-              exit 1
-            fi
+            set -ex
+            rm -rf ./_tmp
+    - change-workdir:
+        title: Switch working dir to test / _tmp dir
+        inputs:
+        - path: ./_tmp
+        - is_create_path: true
+    - git::https://github.com/bitrise-steplib/bitrise-step-simple-git-clone.git@master:
+        inputs:
+          - repository_url: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git # Remote or local URL
+          - branch: master
+          - clone_into_dir: .
+    - path::./:
+        inputs:
+        - xcodebuild_test_options: -verbose
+        - export_uitest_artifacts: "true"
+        - verbose: "yes"
 
-  ci-objc_headless_mode_failing_ui_test:
+  sample_retry_after_test_fail:
```

Comment:
NOTE: I feel like we should have a single `sample` workflow.

I think the `sample` workflow should be used this way:
- You have a bug/need a feature and decide to contribute your changes
- Clone the repository and first ensure the `sample` wf is running, so that you can locally run the step
- Start changing the code and might update the `sample` wf to examine the changes
- When the changes are done, might the modified `sample` wf is a new 2e2 test case, which we move to the `e2e/bitrise.yml` and restore the original sample

Reply:
Removed this wf